challenge_serving_drinks/src/challenge_serving_drinks/sd_states.py

```diff
@@ -0,0 +1,539 @@
+# System
+import random
+import math
+
+# ROS
+import rospy
+import smach
+
+# TU/e Robotics
+import robot_smach_states.util.designators as ds
+
+from hmi import TimeoutException
+from robocup_knowledge import knowledge_loader
+from robot_skills.util.kdl_conversions import frame_stamped, VectorStamped
+from tue_msgs.msg import People
+from Queue import Queue, Empty
+
+# Knowledge
+COMMON_KNOWLEDGE = knowledge_loader.load_knowledge("common")
+
+
+class AskDrink(smach.State):
```

Comment:
This would be much nicer when implemented as a StateMachine in and of itself.

Reply:
Agree

Reply:
Done, needs a bit more cleaning and will be reflected in a new PR